tests/test_key.py

```diff
@@ -100,3 +134,61 @@ def test_scvs_consistency():
     scvs0 = key.scale_chromatic_values
     scvs = [pc.value_in(key) for pc in key.scale]
     assert scvs0 == scvs
+
+
+def test_str():
+    assert str(Key("C")) == "Cmaj"
+
+
+def test_repr():
+    assert repr(Key("C")) == "Key(tonic=C, mode='Major')"
+
+
+def test_inequality():
+    assert Key("C") != "this is not a Key"
+
+
+def test_whole_tone_scale_intervals_fails():
+    with pytest.raises(ValueError, match=r"should contain 7 intervals, not 6"):
+        _scale_intervals([0, 2, 4, 6, 8, 10])
+
+
+def test_blues_scale_intervals_fails():
+    with pytest.raises(ValueError, match=r"strange interval \(not W/H\)"):
+        _scale_intervals([0, 2, 3, 5, 6, 9, 10])
```

Comment:
Wouldn't blues scale be this?
```suggestion
        _scale_intervals([0, 3, 5, 6, 7, 10])
```

Reply:
I think there are different variations, but might make the most sense just to change the name of the test (`test_scale_with_large_interval_fails`?). Your suggested change would fail the `if len(values) != 7:` guard clause, and it wouldn't reach the lines we want to exercise.